modules/RTC/TraceablePeerConnection.js

```diff
@@ -63,10 +84,71 @@ function TraceablePeerConnection(
     this.id = id;
 
     /**
+     * Indicates whether or not this instance is used in a peer to peer
+     * connection.
+     * @type {boolean}
+     */
+    this.isP2P = isP2P;
+
+    // FIXME: We should support multiple streams per jid.
+    /**
+     * The map holds remote tracks associated with this peer connection.
+     * @type {Object<string, Object.<MediaType, JitsiRemoteTrack>>}
+     */
+    this.remoteTracks = {};
+
+    /**
+     * A map which stores local tracks mapped by {@link JitsiLocalTrack.rtcId}
+     * @type {Object.<string, JitsiLocalTrack>}
+     */
+    this.localTracks = {};
+
+    /**
+     * @typedef {Object} TPCGroupInfo
+     * @property {string} semantics the SSRC groups semantics
+     * @property {Array<number>} ssrcs group's SSRCs in order where the first
+     * one is group's primary SSRC, the second one is secondary (RTX) and so
+     * on...
+     */
+    /**
+     * @typedef {Object} TPCSSRCInfo
+     * @property {Array<number>} ssrcs an array which holds all track's SSRCs
+     * @property {Array<TPCGroupInfo>} groups an array stores all track's SSRC
+     * groups
+     */
```

Comment:
at line 114 above it defines TPCSSRCInfo as an {Object} with the properties defined below. Does it answer you question ?